src/modules/auth/actions/use-unlocked-account.js

```diff
@@ -29,6 +30,7 @@ export const useUnlockedAccount = (unlockedAddress, callback = logError) => (dis
       return callback(null)
     }
     dispatch(updateIsLoggedAndLoadAccountData(unlockedAddress))
+    dispatch(loadAccountHistory())
```

Comment:
The account history should be loaded whether the user is using MetaMask or a local Ethereum node.  Suggest putting `dispatch(loadAccountHistory())` on line 20, right after the `loadAccountData` dispatch.